addons/account/models/account_payment_term.py

```diff
@@ -251,7 +251,7 @@ def _compute_terms(self, date_ref, currency, company, tax_amount, tax_amount_cur
     @api.ondelete(at_uninstall=False)
     def _unlink_except_referenced_terms(self):
         if self.env['account.move'].search_count([('invoice_payment_term_id', 'in', self.ids)], limit=1):
-            raise UserError(_('You can not delete payment terms as other records still reference it. However, you can archive it.'))
+            raise UserError(_('Uh-oh! Those payment terms are quite popular and can’t be deleted, as there are still some records referencing them. How about archiving them instead?'))
```

Comment:
```suggestion
            raise UserError(_('Uh-oh! Those payment terms are quite popular and can’t be deleted since there are still some records referencing them. How about archiving them instead?'))
```